### ISSUE TITLE
aria-role doesn't work

### ISSUE THREAD

**Issue by gentlee** [necolas/react-native-web#2592]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the issue

`aria-role` property for TouchableOpacity doesn't work at all.

So have to use `accessibilityRole` instead.

### Expected behavior

`aria-role` works the same as `accessibilityRole`

### Steps to reproduce

Try to render:
```
<TouchableOpacity aria-role='button' />
```

This works:
```
<TouchableOpacity accessibilityRole='button' />
```

### Test case

wil provide later if needed

### Additional comments

Tried latest versions, including "0.19.9"

**Comment by necolas** [comment]
The prop is called `role` not `aria-role`
